src/Wallabag/ApiBundle/Controller/WallabagRestController.php

```diff
@@ -143,8 +149,21 @@ public function postEntriesAction(Request $request)
             $this->assignTagsToEntry($entry, $tags);
         }
 
+        if (!empty($isStarred)) {
```

Comment:
Instead of `!empty(` I would rather check the trueness of the value. Like:

```php
if (true === (bool) $isStarred) {
    $entry->setStarred(true);
}
```

Since `isArchived` & `isStarred` are false by default.